examples/quickstart-pytorch/run.sh

```diff
@@ -2,16 +2,13 @@
 set -e
 cd "$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"/
 
-# Download the CIFAR-10 dataset
-python -c "from torchvision.datasets import CIFAR10; CIFAR10('./data', download=True)"
```

Comment:
How does HF DS handle concurrent requests when the dataset isn't cached on disk yet? We experienced issues with `torchvision.datasets` where concurrent clients trying to download the dataset corrupted each other.

Reply:
In empirical testing, I found it works ok but haven't found that stated explicitly in any docs. I would need to investigate the code or ask a question in the HF forum to make it 100% certain.